src/Home/Home.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { QuoteData } from '../Data';
+import { useEffect } from 'react';
+
+const Home = () => {
+    const randomQuote = Math.floor(Math.random() * Math.floor(QuoteData.length));
+
+    useEffect(() => {
+        document.body.style.backgroundImage =
+          "url('https://source.unsplash.com/1600x900/?')";
+      }); 
+
```

Comment:
why do you need to use this code on every render? Other than that Everything looks great. Clean and readable codes. Great work.